feature-libs/product-configurator/rulebased/components/add-to-cart-button/configurator-add-to-cart-button.component.html

```diff
@@ -41,16 +41,22 @@
           class="cx-btn btn btn-block btn-primary cx-add-to-cart-btn"
           (click)="onAddToCart(container.configuration, container.routerData)"
           [attr.aria-label]="
-            (getButtonResourceKey(container.routerData, container.configuration)
-              | cxTranslate) +
+            (getButtonResourceKey(
+              container.routerData,
+              container.configuration,
+              isQuoteCartActive() | async
```

Comment:
strict mode issue

Reply:
@ChristophHi, I meant this strict mode issue